### ISSUE TITLE
[@types/node]broken document comment

### ISSUE THREAD

**Issue by kgtkr** [DefinitelyTyped/DefinitelyTyped#27890]
This should not be a document comment

![image](https://user-images.githubusercontent.com/17868838/43697981-9110ac60-9982-11e8-851f-dbe1687b82de.png)
![image](https://user-images.githubusercontent.com/17868838/43698000-a918f8e4-9982-11e8-84bf-83e0b0300f89.png)

```ts
/************************************************
*                                               *
*                   XXXXXXX                     *
*                                               *
************************************************/

```

↓

```ts
/*===============================================
*                                               *
*                   XXXXXXX                     *
*                                               *
===============================================*/

```

**Comment by Flarna** [comment]
should be fixed via #28193

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
